app/retail/emails.py

```diff
@@ -787,23 +787,24 @@ def render_start_work_applicant_expired(interest, bounty):
 def render_new_bounty_roundup(to_email):
     from dashboard.models import Bounty
     from external_bounties.models import ExternalBounty
```

Comment:
from external_bounties.models import ExternalBounty

this doesnt exist anymore.. you gotta merge the upstream master in!